source/documentation/data-docs/curated-databases-docs/amazon-athena.md

```diff
@@ -103,52 +139,38 @@ LOCATION 'location';
 
 Here, `format` and `location` are the same as above.
 
-#### Using the Athena UI
-
-Selecting __Create table__ in the database window brings up a menu list with the following options:
-
-*   __Create table__
-    +   From S3 bucket data
-    +   from AWS Glue Crawler
-*   __SQL templates__
-    +   CREATE TABLE
-    +   CREATE TABLE AS SELECT
-
-![](images/curated-databases/athena-create-1.png)
-
-Selecting __CREATE TABLE__ or __CREATE TABLE AS SELECT__ generates an example query that you can edit to create a new table. These example queries are of the same form as those described in the previous section.
-
-Selecting __from S3 bucket data__ will open a new window that guides you through four steps to create a new table from data in S3:
-
-1. Select which database to store your table in, input a name for your table and input the S3 path to the data from which you want to make the table. The S3 path should be of the form `s3://bucket/folder/`.
-2. Select the format of the input data. You may also have to complete additional fields depending on the format of the input data.
-3. Input the name and data type of each column in the table. When adding a large number of columns, it may be easier to use the __Bulk add columns__ option.
-4. Select whether you want to partition the data in the table. This step is optional.
 
 ### Delete a table
 
-To delete a table using the Athena UI, select the three dots (⋮) next to the name of the table you want to delete and select __Delete table__.
+To delete a table using the Athena UI, select the three dots (⋮) next to the name of the table you want to delete and select **Delete table**. 
 
-## Run a query
+### Run a query
 
 To create and run a new query:
 
 1.  Select the plus (+) tab above the editor window.
 2.  Write your code in the editor window (or copy and paste from another editor).
-3.  Select __Run query__.
+3.  Select **Run query**.
 
 Progress information, including the estimated time elapsed will be displayed in the results window while the query is being processed.
 
 Once the query has been completed, the output will be displayed in the results window.
 
-You can save a query (not including the resulting output) by selecting __Save as__ and entering a name and description. Saved queries are available from the __Saved Queries__ tab in the Athena toolbar.
+You can save a query (not including the resulting output) by selecting **Save as** and entering a name and description. Saved queries are available from the **Saved Queries** tab in the Athena toolbar.
```

Comment:
Can you check that you can actually do this from your AP account? I'm not sure if we allow users to do this?